app/helpers/patient_details_helper.rb

```diff
@@ -68,6 +68,9 @@ def comprehensive_details
       american_indian_or_alaska_native: american_indian_or_alaska_native || false,
       asian: asian || false,
       native_hawaiian_or_other_pacific_islander: native_hawaiian_or_other_pacific_islander || false,
+      race_unknown: race_unknown || false,
```

Comment:
so these fields actually need to be added at the end of `comprehensive_details` instead because the order here matters and it's the order that the columns show up in the export
![Uploading Screen Shot 2020-11-17 at 9.40.47 AM.png…]()

Reply:
@timwongj I've moved the orders of those values to the bottom of the list now. Sorry for the delay!